docs/VR/GetSupportedLanguages/assets/GetSupportedLanguages.txt

```diff
@@ -1,44 +1,23 @@
 participant App
```

Comment:
App can be removed from this diagram, since it is not involved in this flow and cannot access this information

Reply:
@jacobkeeler please see updates in ba730b1